### DISCUSSION TITLE
Allow Reactions on the GitHub Public Roadmap

### DISCUSSION THREAD

**Discussion by Danielku15** [community/community#124981] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I was wondering if there is a general interest and a benefit for allowing reactions on the GitHub Public Roadmap items: 

https://github.com/orgs/github/projects/4247/views/1

I understand that GitHub teams also have only limited capacity but sometimes I am a bit surprised about certain decisions around work items. As long proposals are within the discussions here, people can participate indirectly in the prioritization. But as soon they are on the roadmap it is not possible to vote for items.

With voting/reactions enabled people could express interest into work items which might allow a reprioritization. 

One example currently affecting me: The fact that Artifacts v4 for GHES is only planned for Q4 implies a huge cost and time problem to all our in-house development. On every build pipeline we compile 5 minutes, upload 20mins, download 15mins and continue. The benefit of the new artifact backend is huge and was advertised accordingly. These delays will multiply the whole development period across multiple teams and products following this build principle. In comparison hardly any item in Q2 or Q3 has a direct added value comparable to this.

How could now this voting/reaction system help? 

If there would be reactions, people could make visible their needs and GitHub might reconsider their priorities when feasible. 
If many people vote, I see myself confirmed that this has huge value and should be done earlier than planned. 
If not many people vote, I see that the needs on other items seem to be higher and we are maybe in a special position. 
Not having insights in the priorities and decisions, and no way to express the needs, leaves back quite a bit of frustration.

**Comment by ArmSershant** [comment]
Hello @Danielku15, hope you are doing well.
Enabling reactions on GitHub Public Roadmap items could significantly enhance the platform's responsiveness to user needs, improving transparency, engagement, and satisfaction. By allowing users to express their priorities, GitHub can make more informed decisions that align with the community's most pressing needs. This approach can lead to a more balanced and effective development process, ultimately benefiting both GitHub and its users.
